client/layout/masterbar/style.scss

```diff
@@ -286,3 +286,57 @@ $autobar-height: 20px;
 		margin-right: auto;
 	}
 }
+
+.masterbar__recent-drafts.popover {
+	width: 280px;
+
+	.popover__inner {
+		overflow-y: auto;
+		text-align: left;
+		max-height: 70vh;
+
+		.async-load {
+			margin: 16px;
+			width: auto;
+		}
+	}
+}
+
+.masterbar__publish {
+	display: flex;
+
+	.async-load {
+		display: none;
+	}
+}
+
+.masterbar__toggle-drafts.button.is-borderless {
+	background: lighten( $gray, 26% );
+	border-left: 1px solid lighten( $gray, 10% );
+	color: $gray-text-min;
+	height: 36px;
+	margin: 5px 8px 5px -10px;
+	padding: 0 8px;
+	border-radius: 0 3px 3px 0;
+	position: relative;
+
+	.count {
```

Comment:
Seems odd we pull in `<Count />` just to change or remove most all its styles, which appears to be its main purpose for existing. Consider:

- Add optional prop to `<Count />` to display as unstyled
- Don't use `<Count />` at all? `numberFormat` ([in component](https://github.com/Automattic/wp-calypso/blob/300b5ae1adf50620772957b173955034e888ced0/client/components/count/index.jsx#L11)) is available from `localize` higher-order component directly